app/assets/javascripts/battletype/ship.js

```diff
@@ -11,13 +11,35 @@
         }
       },
       velocity: {
-        get: function () { return this._word; },
+        get: function () { return this._velocity; },
         set: function (v) { this._velocity = v; }
       },
       damage: {
         get: function () { return this._damage; },
         set: function (d) { this._damage = d; }
       },
+      height: {
+        get: function () {
+          if (this._height) {
+            return this._height;
+          } else {
+            // FIXME: Find a better way to get this value.
```

Comment:
Why don't we add the ship to the combat (i.e. the DOM) _before_ setting its Y position? It's outside the visible window, so this shouldn't change anything for the player.